src/lib/utils/sendBeacon.spec.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ *
+ * stylelounge.de
+ *
+ * Copyright (C) SNM Style Net Media GmbH
+ * MIT Licensed
+ */
+import {default as send} from "./sendBeacon";
+import * as sinon from "sinon";
+import IManifest from "../types/IManifest";
+declare let global: any;
+describe("sendBeacon ", () => {
+    let manifest: IManifest;
+    before(() => {
```

Comment:
I would remove `before()`, make `manifest` a `const` and assign the value directly, it's the same behavior and less code